src/views/FavoriteFoods.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+
+
+
+class FavoriteFoods extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      favorites: 0
+    }
+  }
+
+
+ addFavoriteCount = () => {
+   this.state({favorites: this.state.favorites + 1});
```

Comment:
You mean:

```js
   this.setState({ favorites: this.state.favorites + 1 });
```

And the most correct way to make state changes that depend on prior state is to pass `setState` a function from the previous state, i.e.

```js
    this.setState(prevState => ({ favorites: prevState.favorites + 1 }));
```

Note you have to wrap the `{ ... }` in "extra" parentheses, otherwise the arrow function thinks the `{` starts a statement block.

Reply:
yes this.setState is correct, I did mean that. @dahlbyk    you mean extra () for the arrow function. check.